flightreservation/Views/Login.js

```diff
@@ -1,11 +1,43 @@
 
-import React from 'react';
-import {View,Image,Text,TextInput,TouchableOpacity,StyleSheet} from 'react-native';
-
+import React,{useState} from 'react';
+import {View,Image,Text,TextInput,TouchableOpacity,StyleSheet,Alert} from 'react-native';
 
+import auth from '@react-native-firebase/auth';
 
 const Login = ({navigation})=>
-{
+{  
+    
+  //-------------------------------------------------------------------------------//  
+  function logMailPassword(email,password)
+  {
+      auth().signInWithEmailAndPassword(email,password)
+      .then(() =>{
+          console.log("Logen succesfull whit", email)
+          navigation.navigate('FlightsView');
+      })
+      .catch(error =>
+        {                      
+            if(error.code === 'auth/user-not-found')
+            {
+                Alert.alert(
+                    'Email No Registered',
+                    'Check The Email Input',
+                    [                      
+                      { text: 'OK'}
+                    ],
+                    { cancelable: false }
+                  );                
+            }                
+        });                 
+  }
+  //----------------------------------------------------------------------------------------//
+  
+
```

Comment:
mentors recommend that we use "prettier" from the visual code to accommodate the code